src/blinky/constr/arty.sdc

```diff
@@ -1 +1 @@
-create_clock -period 10.0 \$abc\$2843\$iopadmap\$clk_i
+create_clock -period 10.0 clk
```

Comment:
What does the XDC file say?

Reply:
The name of the input clock pin is `clk_i`. but the design contains an explicit BUFG which renames the system clock to `clk`.

Currently we do not have yet enabled the XDC parsing plugin yet in fpga-tool-perf. Once the plugin is enabled, we can drop the use of SDC.